data/battle_scripts_1.s

```diff
@@ -437,6 +437,47 @@ gBattleScriptsForMoveEffects::
 	.4byte BattleScript_EffectMortalSpin              @ EFFECT_MORTAL_SPIN
 	.4byte BattleScript_EffectHit                     @ EFFECT_GIGATON_HAMMER
 	.4byte BattleScript_EffectSaltCure                @ EFFECT_SALT_CURE
+	.4byte BattleScript_EffectChillyReception		      @ EFFECT_CHILLY_RECEPTION
+
+BattleScript_EffectChillyReception::
+	attackcanceler
+	printstring STRINGID_PKMNTELLCHILLINGRECEPTIONJOKE
+	waitmessage B_WAIT_TIME_LONG
+	attackstring
+	ppreduce
+	jumpifhalfword CMP_COMMON_BITS, gBattleWeather, B_WEATHER_SUN_PRIMAL, BattleScript_ExtremelyHarshSunlightWasNotLessened
```

Comment:
The user is supposed to still switch out even if the weather can't be changed.

Also, these three jumps appear 8 times in this file. You should make a macro so you don't need to duplicate even more code than already is.

Reply:
This was actually a mistake, but what I wanted to do didn't work anyway... Oops! I've updated the logic for this portion to always switch though. There might still be a better way to do what I did. I'm all ears!

I also went ahead and made a macro for those other instances of this jump even though I don't use it here anymore.